modules/rules_ruby/0.8.0/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: "examples/gem"
+  matrix:
+    bazel: ["7.x", "6.x"]
+    # NB: Windows is removed due to https://github.com/bazel-contrib/rules_ruby/issues/64
+    platform: ["debian10", "macos", "ubuntu2004"]
+  tasks:
+    run_tests:
+      name: "Run test module"
+      platform: ${{ platform }}
+      test_targets:
+        - "//..."
```

Comment:
This block is missing `bazel: ${{ bazel }}`

Reply:
Thank you, I've submitted https://github.com/bazel-contrib/rules_ruby/pull/95 to fix this.